### ISSUE TITLE
Double tap in full mode should switch preso to list mode

### ISSUE THREAD

**Issue by pepelsbey** [shower/shower#104]
The same as `Esc` shortcut, as currently there's no way to exit full mode on mobile devices.

**Comment by toptalo** [comment]
Hello. What about swipe Up to exit full mode? Is this gesture used anywhere in shower?

**Comment by pepelsbey** [comment]
@toptalo, swipe up could be used, there are no gestured used in Shower, only forward/backward taps.

**Comment by toptalo** [comment]
@pepelsbey, what if replace tap action to swipe?
Swipe left/right for backward/forward and swipe up to exit full mode? Or make swipe only for exit?

**Comment by pepelsbey** [comment]
It makes sense only when interface is responding to your swipes, otherwise it feels like it’s stuck. Taps work better in this case.

**Comment by toptalo** [comment]
Ok. How can I try to add this swipe to exit action?
1. clone shower/touch
2. create a branch
3. make changes
4. create pull request in shower/touch?

**Comment by pepelsbey** [comment]
@toptalo, exactly. You can have a look at what's currently going on in [shower/core PRs](https://github.com/shower/core/pull/120).

**Comment by shvaikalesh** [comment]
@pepelsbey Should we close this one in favor of #344 or let's have them both?
